R/scale-.r

```diff
@@ -555,6 +560,22 @@ ScaleContinuous <- ggproto("ScaleContinuous", Scale,
     self$rescaler(x, from = range)
   },
 
+  get_limits = function(self) {
+    if (self$is_empty()) {
+      return(c(0, 1))
+    }
+
+    if (is.null(self$limits)) {
+      self$range$range
+    } else if (is.function(self$limits)) {
+      # if limits is a function, it expects to work in data space
+      self$trans$transform(self$limits(self$trans$inverse(self$range$range)))
+    } else {
+      # NA limits for a continuous scale mean replace with the min/max of data
+      ifelse(is.na(self$limits), self$range$range, self$limits)
+    }
+  },
+
```

Comment:
Is this just the same as `Scale$get_limits()`?

Reply:
Can you test if it breaks anything in ScaleDiscrete? I don’t like having such a small method overwriting for no reason